media-tv/mythtv/mythtv-29.1-r2.ebuild

```diff
@@ -63,17 +62,14 @@ COMMON="
 		sys-fs/udisks:2
 	)
 	cec? ( dev-libs/libcec )
-	dvb? (
-		virtual/linuxtv-dvb-headers
-	)
 	dvd? (
 		dev-libs/libcdio:=
 		sys-fs/udisks:2
 	)
 	egl? ( media-libs/mesa[egl] )
 	fftw? ( sci-libs/fftw:3.0= )
 	hls? (
-		<media-libs/libvpx-1.7.0:=
+		media-libs/libvpx:=
```

Comment:
I hope you are sure that it works, didnt I ask you to check this during the initial review? :)

Reply:
mythtv-29.1-r1 (original) is not compatible with the stable tree. Change to <media-libs/libvpx-1.8.0:= and it builds. Put same restriction in mythtv-30.0_p20190808.